src/Chapter09/Chapter09.csproj

```diff
@@ -1,11 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProductName>Chapter09</ProductName>
+    <ExpectedWarnings>$(ExpectedWarnings)</ExpectedWarnings>
+    <!-- CS0178: Invalid rank specifier: expected ',' or ']' -->
+    <!-- CS1513: } expected -->
+    <!-- CS1514: { expected -->
+    <!-- CS0826: No best type found for implicitly-typed array -->
+    <IgnoredWarnings>$(IgnoredWarnings),CS0178,CS1513,CS1514,CS0826</IgnoredWarnings>
   </PropertyGroup>
   <Import Project="..\Chapter.props" />
+  <ItemGroup>
+    <Compile Remove="Listing09.13.RecordStructInitialization.cs" />
+  </ItemGroup>
   <ItemGroup>
     <Compile Include="..\Shared\Program.cs">
       <Link>Program.cs</Link>
     </Compile>
   </ItemGroup>
-</Project>
+  <ItemGroup>
+    <None Include="Listing09.13.RecordStructInitialization.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="NPOI" Version="2.6.1" />
```

Comment:
Are we using something from the NPOI library? The book contents aim to avoid taking dependencies on outside libraries (with the exception of the tests)

Reply:
Has this comment been addressed as well as the warning excludes?